Src/Support/Google.Apis.Auth/OAuth2/GoogleAuthConsts.cs

```diff
@@ -172,6 +172,18 @@ public static class GoogleAuthConsts
                     response.StatusCode == HttpStatusCode.ServiceUnavailable, // 503
             }));
 
+        /// <summary>
+        /// If <paramref name="originalPolicy"/> has the <see cref="ExponentialBackOffPolicy.RecommendedOrDefault"/> flag, the returned value
+        /// will be identical to <paramref name="originalPolicy"/> except it won't have the <see cref="ExponentialBackOffPolicy.RecommendedOrDefault"/> or
+        /// <see cref="ExponentialBackOffPolicy.UnsuccessfulResponse503"/> flags. Otherwise, the returned value will be <paramref name="originalPolicy"/>.
+        /// This method facilitate honoring any custom policies that may have been set. Calling code is resposible for honoring the recommended policies.
+        /// </summary>
+        internal static ExponentialBackOffPolicy StripOAuth2TokenEndpointRecommendedPolicy(ExponentialBackOffPolicy originalPolicy) =>
```

Comment:
I'm very glad this is internal, as it feels a bit odd to me. I'm not quite sure what the purpose is yet, but I suspect I'll understand later.

Reply:
We just don't have enums for configuring retries, they are also Flags.

The recommended OAuth Token endpoint retry includes retrying on 503, but not on exceptions. So, if the original enum value has RecommendedOrDefault set we don't need to add a specific policy for 503. We only keep here what's not recommended, in that way, calling code know whether it needs to add a policy different than recommended.
Calling code also needs to check if it needs to add the recommended policy or not.

Happy to chat about this though. I had several different approaches for this, but ended up preferring this one.